tests/debugger/oegdb/enc/contract.c

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/internal/error.h>
+#include <openenclave/internal/sgxtypes.h>
+#include <openenclave/internal/tests.h>
+#include <stdio.h>
+#include <string.h>
+#include "../../../../enclave/core/sgx/td.h"
+
+// These values are filled by the debugger.
+// The variables are marked volatile so that the compiler does not hard-code
+// the initialization value of the variables in comparision.
+// Since 0 could be a legal value for some of these items, they are
```

Comment:
This could probably use some additional elaboration as to why they need to be illegal values. (i.e. expect the test evaluation of the variables to fail if commands.gdb could not overwrite the variables in the debugger)

Reply:
Fixed.